description.ext

```diff
@@ -6,13 +6,9 @@ overviewPicture = "images\Overview.jpg";
 disabledAI = 1;
 
 // Respawn + Revive
-respawn = 3; //BASE
-RespawnDelay = 30;
-respawnTemplates[] = {"Revive","Wave","Spectator","Count"};
+//respawn = 3; //BASE
```

Comment:
Uncomment this https://community.bistudio.com/wiki/Arma_3_Respawn BASE means spawn on marker (BIS_fnc_addRespawnPosition adds a marker just like that)